source/views/building-hours/detail/index.js

```diff
@@ -98,10 +107,22 @@ export class BuildingHoursDetailView extends React.PureComponent<Props, State> {
 
 function mapStateToProps(state: ReduxState): ReduxStateProps {
   return {
-    favoriteBuildings: state.buildings ? state.buildings.favoriteBuildings : [],
+    favoriteBuildings: state.buildings.favoriteBuildings
```

Comment:
We know that if state.buildings exists, then favoriteBuildings will too; it's the buildings key itself that may not exist (this line was correct before [unless something was yelling about it in which c add I am probably wrong])